### DISCUSSION TITLE
Show the device a tweet was posted from

### DISCUSSION THREAD

**Discussion by deepwaterspark** [insin/control-panel-for-twitter#210] [Ideas]
Please bring back the feature where you could see the device the poster was using to post their tweets on? It didn't need to be removed but rather be a setting to be hidden from users that didn't like it and not be forced upon the whole website. Thank you.

**Comment by marypcbuk** [comment]
twitter chose to hide this but it seems to still be in the metadata, but not clear if an extension could extract it

**Reply by deepwaterspark** [reply]
It still would mean a lot, as it didn't deserve to be removed.

**Reply by marypcbuk** [reply]
You do know this is the repo for an unofficial third party extension that can't control what twitter chooses to do?
